src/modbus_proxy.py

```diff
@@ -132,6 +132,7 @@ def __init__(self, config):
         self.modbus_port = url.port
         self.timeout = modbus.get("timeout", None)
         self.connection_time = modbus.get("connection_time", 0)
+        self.unit_id_remapping = config.get("unit_id_remapping", None)
```

Comment:
we could have the default value be `{}` so we don't have to check for None on every call to `_transform_request`

Reply:
Done. I _do_ have to make it this, though:

```py
self.unit_id_remapping = config.get("unit_id_remapping", {})  or {}
```

... because if `unit_id_remapping` is present in YAML, but empty, it appears to still be set to `None` despite the `{}` default in `get`.

Reply:
awesome 👍🏽